http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20200415_drop_notebookServiceAccount.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="leonardo" author="qi" id="drop_notebookServiceAccount">
+        <dropColumn tableName="CLUSTER" columnName="notebookServiceAccount"/>
+        <addNotNullConstraint columnDataType="varchar(254)"
```

Comment:
Just curious, are there any records in prod where `clusterServiceAccount` is null?

Reply:
yea. there are 50 clusters that have clusterServiceAccount being null, and notebookSA being non null. But they're all `Deleted`, and most of them are test clusters from `nobody@unset.com`, and a few from Hussein, `tmajaria@broadinstitute.org`, `dsiedzik@broadinstitute.org`